plugins/docker/docker_

```diff
@@ -45,44 +46,179 @@ https://docs.docker.com/compose/reference/envvars/):
     env.DOCKER_HOST unix://var/run/docker.sock
 
 =back
+
+=head1 AUTHORS
+
+This section has been reverse-engineered from git logs
+
+* Codimp <contact@lithio.fr>: original rewrite
+* Rowan Wookey <admin@rwky.net>: performance improvement
+* Olivier Mehani <shtrom@ssji.net>: Network support, ClientWrapper, gerenal
+  cleanup
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf suggest
+
 """
 
 import os
 import sys
-import docker
+from functools import cached_property
 from multiprocessing import Process, Queue
 
+# Early autoconf-style detection
+try:
+    import docker
+except ImportError as e:
+    print(f'no ({e})')
```

Comment:
I think, you can *translate* the pythonic error message here into a user friendly text: `no (failed to import python module 'docker')`.

Reply:
I could, but I went down a different path now, doing all the autoconf checks at once, so it's more difficult to do now.